bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/ConfigDescriptionParameter.java

```diff
@@ -362,35 +371,35 @@ public String getContext() {
      *         false
      */
     public boolean isRequired() {
-        return this.required;
+        return required;
     }
 
     /**
      * Returns the default value of the configuration parameter.
      *
      * @return the default value of the configuration parameter (could be null)
```

Comment:
The annotations now provide ths info so please remove this redundant info from the impacted JavaDocs:

```suggestion
     * @return the default value of the configuration parameter
```

Reply:
@wborn Done :+1: